package.nls.hindi.json

```
@@ -0,0 +1,31 @@
+{
+    "python.command.python.sortImports.title": "आयात क्रमबद्ध करें",
+    "python.command.python.startREPL.title": "REPL शुरू करें",
+    "python.command.python.createTerminal.title": "टर्मिनल बनाएं",
+    "python.command.python.buildWorkspaceSymbols.title": "वर्कस्पेस सिंबल बनाएं",
+    "python.command.python.runtests.title": "सभी टेस्ट चलाएं",
+    "python.command.python.debugtests.title": "सभी टेस्ट डीबग करें",
+    "python.command.python.execInTerminal.title": "टर्मिनल में पायथन फाइल चलाएं",
+    "python.command.python.setInterpreter.title": "इंटरप्रेटर चुनें",
```

```suggestion
    "python.command.python.setInterpreter.title": "इंटरप्रिटर चुनें",
```
Interpreter is phonetically `ɪnˈtɜrprətər` sounds closer to इंटरप्रिटर than इंटरप्रेटर . This is not blocking.
/cc @karrtikr any opinions here?

---

I think `प्रि` translates to `pri` instead of `pre` in `interpreter`. `इंटरप्रेटर` sounds closer to me.